api/config/services.yaml

```diff
@@ -41,3 +41,9 @@ services:
         decorates: App\Repository\TopBook\TopBookDataRepository
 
     App\Repository\TopBook\TopBookDataInterface: '@App\Repository\TopBook\TopBookDataRepository'
+
+    # Swagger doc customisation
+    App\Swagger\SwaggerDecorator:
+        decorates: 'api_platform.swagger.normalizer.api_gateway'
+        #arguments: ['@api_platform.swagger.normalizer.api_gateway.inner' ]
+        #autoconfigure: false
```

Comment:
Maybe the `autoconfigure` is useful here. You don't want to use this normalizer for other normalizations.

Reply:
Ok, fixed. Thanks for the review @alanpoulain :)